neurovault/api/serializers.py

```diff
@@ -229,8 +230,7 @@ class Meta:
 class CollectionSerializer(serializers.ModelSerializer):
     url = HyperlinkedImageURL(source='get_absolute_url', read_only=True)
     owner = serializers.ReadOnlyField(source='owner.id')
-    images = ImageSerializer(many=True, source='image_set')
-    nidm_results = NIDMResultsSerializer(many=True, source='nidmresults_set')
+    images = ImageSerializer(many=True, source='basecollectionitem_set')
```

Comment:
Actually `items` field  of `CollectionSerializer` is overridden in `CollectionViewSet` so this change is not reflected anywhere. Maybe we should completely remove it in the future.